Sources/MapboxMaps/Snapshot/Snapshotter.swift

```diff
@@ -96,6 +96,8 @@ public class Snapshotter {
                 return
             }
 
+            sendTurnstileEvent()
```

Comment:
Can you explain why you chose to send the turnstile here instead or say at the beginning of the method or all the way at the end prior to invoking the completion block with success?

Reply:
This probably should be on line 107, actually. I intended to put it after the guard statements that resulted in failures.